notebooks/energy_volume_curve.ipynb

```diff
@@ -30,14 +30,28 @@
     "ExecuteTime": {
```

Comment:
<p>I'm afraid I don't see how to do a github "suggestion" in NB Review, so these will all be old-school comments!</p><p> </p><p>Nit: "allows to [verb]" is a deutschism. "Allows" is a super funny verb that requires an extra noun to work when(ever? I think. Proof by lack of imagination) combined, i.e. "seatbelts allow drivers to not die in low-speed crashes", "the energy volume curve allows us to calculate...", etc. I don't know all the grammatical terms, but <a href="https://english.stackexchange.com/questions/85069/is-the-construction-it-allows-to-proper-english" target="_blank">here</a> is an interesting english.stackexchange commentary on the topic.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron/pull/1479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
ok I just realized that I didn't even write this text XD